Automattic/woocommerce-payments
# Add Connect Page A/B Test

Fixes #1612 #1436

#### Changes proposed in this Pull Request

Adds an A/B test comparing the original Connect Page with a new design implemented in PR #1600.

The new design has been changed slightly since #1600, with the following changes:

- Terms of Service was added.
- A wrapping div `.connect-account-page-variant` is used to prevent style rules clashing.
- Payments icons are changed per https://github.com/Automattic/woocommerce-payments/issues/1436#issuecomment-824973580







#### Testing instructions

Ensure connected to a8c proxy. Without it, a networking error occurs in the console.

With a new installation of WC Pay,

* Navigate to Payments
* See that a random design for the Connect page is displayed (original vs new). To confirm this, switch the components in the `defaultExperience` and `treatmentExperience` props in `<Experiment />`.

-------------------
